source/includes/fact-match-dump-restore-versions.rst

```diff
@@ -1,9 +1,26 @@
 When using :binary:`~bin.mongorestore` to load data files created by
-:binary:`~bin.mongodump`, be sure that you are restoring to the *same
-major version* of the MongoDB Server that the files were created from.
-For example, if your dump was created from a MongoDB Server running
-version ``4.4.x``, be sure that the MongoDB Server you are restoring to
-is also running version ``4.4.x``.
+:binary:`~bin.mongodump`, be sure that the deployment you are restoring 
+to has either:
+
+- The same major version.
+- The same feature compatibility version. 
+
+For example, if your dump was created from a MongoDB deployment running
+version ``4.4``, be sure that the MongoDB deployment you are restoring 
+to is also running version ``4.4`` or has the FCV level set to 
+``4.4``. 
+
+See :dbcommand:`setFeatureCompatibilityVersion` for 
+information on changing feature compatibility versions.
+
+.. note::
+
+   The BSON files generated from ``mongodump`` are restorable 
+   into MongoDB deployments starting with the version they were created 
+   on and in all future versions.
+
+   This guarantee does not apply to metadata, archive, or oplog replay 
+   files.
```

Comment:
[question]

Is there any additional context we can provide here? What happens if I try to restore these types of files? Are they just not restored? Does the entire restore operation fail?

Reply:
Updated this with some input from Tim, the results are largely undefined, but I put some of the common possibilities that could result if attempted.